packages/snaps-sdk/src/jsx/components/Container.ts

```diff
@@ -2,13 +2,19 @@ import type { GenericSnapElement } from '../component';
 import { createSnapComponent } from '../component';
 import type { FooterElement } from './Footer';
 
+/**
+ * Definition of container background colors.
+ */
+export type ContainerBackgroundColor = 'default' | 'alternative';
```

Comment:
Maybe somewhat confusing that the default color is `alternative` and `default` is the actual alternative color for the Snaps backgrounds 😄 

Not sure exactly how we wanna name these

Reply:
It's fine to use `default` and `alternative`, but I feel like `default` should be the default in the Snaps API 😅 We can swap them around in the extension implementation.

Reply:
It's a bit confusing but perhaps the right way to do it. I definitely don't want "default" to not be the default 😂

Reply:
Yeah, I think so.

Reply:
Yep!